setup.py

```diff
@@ -29,6 +24,7 @@
         packages=["fs_gcsfs"],
         license="MIT",
         python_requires=">=3.5",
+        setup_requires=['setuptools_scm'],
```

Comment:
We'll need to add the new build-time dependency on setuptools_scm in the corresponding meta.yml file as well.

Reply:
yes, the conda-forge bot will automatically issue a PR to the feedstock that we can take over